### PR TITLE
Add feedback instruction to help page

### PR DESCRIPTION
**Pull request by simoneduca**
Describe your changes.
Add basic feedback documentation.

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch? https://add-howto-feedback.pfe-preview.zooniverse.org/
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

## Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
If feedback is still an experimental feature, I'm not sure we should make it public on the help pages.

**Comment by simoneduca** [comment]
Good point, I forgot it's still experimental. Could you suggest anywhere else? https://github.com/zooniverse/Panoptes-Front-End/issues/3858#issuecomment-324928568

File: app/pages/lab/help/how-to-page.cjsx [hunk 1]
```diff
@@ -244,6 +244,46 @@ counterpart.registerTranslations 'en',
       You can create a manifest file in a simple text editor (such as TextEdit or Notepad), although this method is prone to errors like missed or extra commas. People often find it easiest to create manifest files using spreadsheet software such as Google Sheets, iWork Numbers or Microsoft Excel. Creating and maintaining a manifest using a spreadsheet makes the manifest easy to read, and you can export it to CSV format when you're ready to upload your subjects. You can also open existing CSV files in spreadsheet software.
 
       *Note:* if you have a large subject set it may be cumbersome to manually create a manifest. We suggest using a command-line or other tool to copy-paste a directory list of files into a spreadsheet to help you get started.
+
+    ### Generalised Feedback
+
+      Feedback is enabled per-task - there's a checkbox under the answer editing box that toggles it on and off.
+
+      ![screen shot 2017-04-13 at 14 29 36](https://cloud.githubusercontent.com/assets/2725841/25006659/b8d81398-2055-11e7-8cb6-8c660b4173b7.png)
```
Comment (eatyourgreens) [inline app/pages/lab/help/how-to-page.cjsx]: The alt text here ('screen shot 2017-04-13 at 14 29 36') should describe the content of the image. Maybe a short description of what the screenshot shows?

File: app/pages/lab/help/how-to-page.cjsx [hunk 2]
```diff
@@ -244,6 +244,46 @@ counterpart.registerTranslations 'en',
...
+
+      Once enabled, you can add a feedback type by hitting the button:
+
+      ![screen shot 2017-04-13 at 14 30 40](https://cloud.githubusercontent.com/assets/2725841/25006686/d02e5f20-2055-11e7-8647-83bc343cb1fe.png)
```
Comment (eatyourgreens) [inline app/pages/lab/help/how-to-page.cjsx]: Ditto re the image alt text.

File: app/pages/lab/help/how-to-page.cjsx [hunk 3]
```diff
@@ -244,6 +244,46 @@ counterpart.registerTranslations 'en',
...
+      foo.jpg,"FOO","","",
+      bar.jpg,"BAR","","",
+      baz.jpg,"BAZ","this is a custom success message","this is a custom failure message",
+      ```
+
+      #### Feedback summary
+
+      The summary is generated at the end of a classification, based on the subject feedback fields, the active rules, and the task:
+
+      ![screen shot 2017-04-13 at 14 41 11](https://cloud.githubusercontent.com/assets/2725841/25007130/46a18cda-2057-11e7-8215-3c2ceeb0a2bc.png)
```
Comment (eatyourgreens) [inline app/pages/lab/help/how-to-page.cjsx]: Alt text again.

File: app/pages/lab/help/how-to-page.cjsx (hunk 1 above)
Comment (eatyourgreens) [reply app/pages/lab/help/how-to-page.cjsx]: Should the images be added to PFE, instead of pulled in from github?

**Comment by eatyourgreens** [comment]
I don't know. Ideally it would be available to project owners who are using the feature, but I'm not sure how we'd check that.

**Comment by simoneduca** [comment]
Perhaps it should be in the project builder? But I'm not sure if it actually adds any extra useful info to what it's already in there.

**Comment by eatyourgreens** [comment]
Me neither. Another possible place is to use it is as the first pass at a readme for `/app/classifier/feedback`.

**Pull request closed without merging**